app/src/main/java/com/example/chrono/main/timer/CircuitCreate.kt

```diff
@@ -182,4 +184,36 @@ class CircuitCreate : BaseActivity() {
             }
         }
     }
+
+    // Floor to a multiple of timeChangeVal
+    private fun floorVal(valToFloor: Int): Int {
+        return (valToFloor / timeChangeVal) * (timeChangeVal)
+    }
+
+    //When decrementing set time, subtract to the closest multiple of timeChangeVal
+    private fun roundTimeDown(valToRound: Int): Int {
+        return if (valToRound % timeChangeVal == 0) {
+            valToRound - timeChangeVal
+        } else {
+            floorVal(valToRound)
+        }
+    }
+
+    //Change SetNum textView text
+    private fun setSetNum(newVal: Int) {
+        bind!!.setNum.setText(newVal.toString())
+    }
+
+    //Change SetRestTime textView text
+    private fun setRestTime(newTime: Int) {
+        bind!!.setRestTime.setText(newTime.toString())
```

Comment:
Kotlin doesn't use setters/getters, I'm assuming this is throwing a warning. Should be:
bind!!.setRestTime.text = newTime.toString()

Reply:
Also one liner function

Reply:
Yeah that's what I though but it was like this so I didn't end up changing. Im going to fix it up now.

Reply:
can't be changed because TextInputEditText.text requires a variable of type required, doesn't accept strings. Need to use a setter in this case. No warnings are thrown.